test/test_webhook.py

```diff
@@ -35,6 +36,65 @@ def test_it_updates_and_returns_204_if_a_profile_is_found(test_client: FlaskClie
     assert profile.name.preferred == 'Given Names Family Name'
 
 
+# pylint: disable=invalid-name
+def test_it_prevents_unique_constraint_error_when_inserting_existing_email_address(
+        test_client: FlaskClient, webhook_payload: str) -> None:
+    """
+    This test was added as a result of this issue:
+    https://github.com/elifesciences/issues/issues/4633
+    """
+
+    profile_1 = Profile('a1b2c3d4', Name('Foo Bar'))
+    profile_1.add_email_address('1@example.com', restricted=False)
+
+    profile_2 = Profile('b2c3d4e5', Name('Old Name'), '0000-0002-1825-0097')
+
+    db.session.add(profile_1)
+    db.session.add(profile_2)
+    db.session.commit()
+
+    with requests_mock.Mocker() as mocker:
+        mocker.get('http://www.example.com/api/v2.1/0000-0002-1825-0097/record',
+                   json={
+                       'person': {
+                           'name': {
+                               'family-name': {'value': 'Name'},
+                               'given-names': {'value': 'New'}
+                           },
+                           'emails': {
+                               'email': [
+                                   {'email': '1@example.com',
+                                    'primary': True,
+                                    'verified': True,
+                                    'visibility': 'PUBLIC'}
+                               ]
+                           }
+                       }
+                   })
+
+        response = test_client.post('/orcid-webhook/{}'.format(webhook_payload))
```

Comment:
`webhook_payload` is opaque here (the test code doesn't tell us which ORCID it corresponds to); maybe it can be forced to be about `0000-0002-1825-0097`

Reply:
It's a serialised version of `0000-0002-1825-0097` https://github.com/elifesciences/profiles/blob/develop/test/conftest.py#L204. This pattern is used throughout the test suite.